config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+Q0NZBPwCK18TE0EO8hQD4Xbi65F1s2lvBFPv9LvgqbfOEKkPEoWsVIYpQW3BD6SpwlJdbvnMoHhOrkz0ugeHzkMp/20XcwER0yvkMbEWg5fsgmNFvsBaOkF5Hct1oqpSOYVm3ZEFlSDB7YX1YXAqHfwpEqY11q4p+SHUFQM9d4MRB13XNFWwAbFGflnb2oNU/w4aDFNw6ovc0+FsaYxve25oJdU0REWL6Tt2ONaNV842m+nMAcXd2pqIPqfQqS7nOVhTMHbsCrRfTQFvCwV19kE2f6sE8qTUkR9FXHij2bJlDrSzrfOXcXVr/do05U/ntV21ESuTs8vNn8OjBJ5T3TMncWyFrPkRq4IDW/hF2Mny03UM8/nptTa5DTYP8/JY67bk4z/byVzEOOamOmI5uu6UUIVENIaV3y60--NOSBNFn50kH+PZtZ--/MSSStmr1n+64mgZW0L7aQ==
```

Comment:
I apologize, I failed to find this in https://github.com/pulibrary/pdc_describe/blob/main/config/credentials.yml.enc 👍